dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart

```diff
@@ -14,14 +14,14 @@ import 'package:intl/intl.dart' as intl;
 import 'stock_strings_en.dart';
 import 'stock_strings_es.dart';
 
-/// Callers can lookup localized strings with an instance of StockStrings returned
-/// by `StockStrings.of(context)`.
+/// Callers can lookup localized strings with an instance of StockStrings
```

Comment:
The changes in this file look unrelated to updating the lock files. Should it go in a separate PR?